### DISCUSSION TITLE
Authors Missing openreview IDs

### DISCUSSION THREAD

**Discussion by KennethEnevoldsen** [embeddings-benchmark/mteb#1244] [General]
The following authors are missing an openreview ID. This is required for the registering authorship during paper submission:

@GabrielSequeira
@achibb 
@izhx
@awinml
@shreeya-dhakal
@akshita-sukhlecha

I will still add all author names to the paper for the submission but would prefer to register it in the system as well (seems to be required)

**I have added one for (based on a search on openreview):**

@imenelydiaker
@MathieuCiancone

**Comment by achibb** [comment]
Hi thanks I created one, at least I tried. Will take up to 2 weeks they said to get it registered. Else feel free to go on without registration worst case.

~Aaron_Chibb1 would then be the name. Else I can contact them maybe to speed it up?

All the best

________________________________
Von: Kenneth Enevoldsen ***@***.***>
Gesendet: Donnerstag, September 26, 2024 10:49 PM
An: embeddings-benchmark/mteb ***@***.***>
Cc: achibb ***@***.***>; Mention ***@***.***>
Betreff: [embeddings-benchmark/mteb] Authors Missing openreview IDs (Discussion #1244)


The following authors are missing an openreview ID. This is required for the registering authorship during paper submission:

@GabrielSequeira<https://github.com/GabrielSequeira>
@achibb<https://github.com/achibb>
@izhx<https://github.com/izhx>
@awinml<https://github.com/awinml>
@shreeya-dhakal<https://github.com/shreeya-dhakal>
@akshita-sukhlecha<https://github.com/akshita-sukhlecha>

I will still add all author names to the paper for the submission but would prefer to register it in the system as well (seems to be required)

I have added one for (based on a search on openreview):

@imenelydiaker<https://github.com/imenelydiaker>
@MathieuCiancone<https://github.com/MathieuCiancone>

**Reply by KennethEnevoldsen** [reply]
I can see that I can still add you in the system so this is perfectly fine

**Reply by achibb** [reply]
Ok if needed I can add the names and institutions mentioned here to the authors in overleaf if it helps? But don’t want to distort any orders, so whatever helps you!

**Comment by akshita-sukhlecha** [comment]
Hi, 
I too have registered now and it says it will take up to 2 weeks for profile activation. My ID: ~Akshita_Sukhlecha1

**Reply by KennethEnevoldsen** [reply]
I can see that I can still add you in the system so this is perfectly fine

**Comment by izhx** [comment]
Hi, my ID:  ~Xin_Zhang15

Thanks!!!

**Comment by awinml** [comment]
Hi,

I too have created an openreview account and it says it will take up to 2 weeks for activation. The ID is: ~Ashwin_Mathur1.
Will update here as soon as I get confirmation for activation.

Thanks!

**Reply by KennethEnevoldsen** [reply]
I can see that I can still add you in the system so this is perfectly fine

**Reply by awinml** [reply]
@KennethEnevoldsen My profile (https://openreview.net/profile?id=~Ashwin_Mathur1) just got activated, so shouldn't be any problem now. Thanks!

@achibb @akshita-sukhlecha emailing them at [info@openreview.net](mailto:info@openreview.net) with proof of your credentials will expedite the process.

**Comment by shreeya-dhakal** [comment]
I have just created an account- which would take 2 weeks to be active but here is my openreviewID: ~Shreeya_Dhakal1
@KennethEnevoldsen
